Automattic/jetpack
# Migration plugin: refactor migration component

## Proposed changes:

- Exported existing `ConnectScreenLayout` component from the `js connection package`
- Introduced Migration component, which replaced the Jetpack logo and styling with WordPress.
- Adjusted the admin page to consume the Migration component.
For more details, see the screenshot below.

Closes https://github.com/Automattic/dotcom-forge/issues/1491

## Testing instructions:
* Launch a [JN site with the migration plugin](https://jurassic.ninja/create/?jetpack-beta&branches.jetpack-migration=update/migration-plugin&wp-debug-log).
* Check if there is a new WordPress.com branding style
* Click on the `Get Started` button and check if everything works as before this PR

## Does this pull request change what data or activity we track or use?
No

## Screenshot
<img width="1112" alt="Screenshot 2023-02-06 at 10 21 33" src="https://user-images.githubusercontent.com/1241413/217011592-49535d65-4d2a-41d1-bdc6-10c9dccca7ee.png">
